### ISSUE TITLE
--audio-format vorbis don't work

### ISSUE THREAD

**Issue by starinspace** [yt-dlp/yt-dlp#1836]
### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2021.11.10.1**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Description

I used this script with youtube-dl before, and it work (sort of, its slow and crashes and not good support anymore), however, I did try this with yt-dlp, and it does download, but it does not convert to ogg. I get **ERROR: 'libvorbis'**. I do have ffmpeg and it does work with youtube-dl when trying (however slow (ca 40kbit/s), with yt-dl its mbit and crashes).

`for %%a in (%*) do (`
`yt-dlp.exe ^`
`--extract-audio ^`
`--audio-format vorbis ^`
`--output "Youtube-OGG/%%(playlist_title)s/%%(playlist_index)s - %%(title)s.%%(ext)s" ^`
`%%a`
`)`

### Verbose log

```shell
[debug] ffmpeg command line: ffprobe -show_streams "file:Youtube-OGG\Album - So Far\1 - So Far.webm"
ERROR: 'libvorbis'
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1311, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1349, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1399, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2430, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 2920, in process_info
  File "yt_dlp\YoutubeDL.py", line 3079, in post_process
  File "yt_dlp\YoutubeDL.py", line 3022, in run_pp
  File "yt_dlp\postprocessor\common.py", line 22, in run
  File "yt_dlp\postprocessor\common.py", line 112, in wrapper
  File "yt_dlp\postprocessor\ffmpeg.py", line 468, in run
  File "yt_dlp\postprocessor\ffmpeg.py", line 399, in _quality_args
KeyError: 'libvorbis'
```

**Comment by pukkandan** [comment]
Same issue as #1645, fixed in 39c04074e7e108bc6e36f3a34ef08a163663144a

**Comment by starinspace** [comment]
> Same issue as #1645, fixed in [39c0407](https://github.com/yt-dlp/yt-dlp/commit/39c04074e7e108bc6e36f3a34ef08a163663144a)

Do you know when it will be released with the fix?
